bin/tg-16

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
+
+/usr/share/steam-buddy/launcher tg-16 mednafen_pce_fast "$@"
```

Comment:
there is also `libretro-mednafen-supergrafx-git`, not sure how to evaluate which is better. I saw some recommendations online of supergrafx. supergrafx also seems to have support for turbografx cd.